requirements.txt

```diff
@@ -9,7 +9,7 @@ channels
 channels_redis
 django_ical
 xlsxwriter
-python_ldap==3.1.0
+python_ldap==3.2.0
```

Comment:
Why change this? What does this have to do with `tools`?

Also, any version `>=3.0` supports Python 3 and should work.

Reply:
It probably doesn't have anything to do with `tools`, but it's a welcome fix nonetheless, because `v3.1.0` isn't available anymore from the [officially linked](https://www.python-ldap.org/en/latest/installing.html#windows) [site](https://www.lfd.uci.edu/~gohlke/pythonlibs/#python-ldap) storing the pre-built binaries for Windows; `v3.2.0` is, though 🙂

Reply:
Should probably be a separate PR though

Reply:
True, but as it's already part of the PR and it's such a small change, I would personally not bother removing it.

Reply:
I agree that a separate PR is better. It would have been okay if it was only part of this one, but it is also a separate change for the same thing in #245. So I just made #247 for simplicity.